lib/health.js

```diff
@@ -155,16 +154,18 @@ const cmdExec = (
       return;
     }
     const lines = res.split(rxLine);
-    let i, ilen, j, jlen, line, header, col, row;
-    for (i = 0, ilen = lines.length; i < ilen; i++) {
-      line = lines[i];
+    const ilen = lines.length;
+    for (let i = 0; i < ilen; i++) {
+      let header, col, row;
+      let line = lines[i];
       if (line) {
         line = line.replace('Use%', 'Use').replace('Mounted on', 'MountedOn');
         col = line.split(process.isWin ? rxCSV : rxSPC);
         if (!header) header = col;
```

Comment:
Sorry, what did you mean?

Reply:
There was a mistake in my comment above, what I wanted to say is the other branch (`else` branch) of this `if` statement will never be executed because you are declaring new `header` variable on each loop iteration and there is no value assigned to it before this point, meaning that on this line `header` is always equal to `undefined`. 

That being said, this code won't work in the same way as before.

Reply:
will it work if I declare a header before the loop?

Reply:
What I need to do that code will work again?